packages/webrtc-firebase/src/webrtc_firebase_proxy.ts

```diff
@@ -135,29 +183,36 @@ class WebRTCFirbase extends Base implements IVideoCall {
     }
   }
 
-  getPeerConnection() {
+  getPeerConnection = () => {
     return this.peerConnection;
-  }
+  };
 
   setup = async () => {
     console.log("Setup webrtc");
-
     try {
-      this.peerConnection = new RTCPeerConnection(peerConstraints);
+      const peerConnection = new RTCPeerConnection(peerConstraints);
+
+      this.peerConnection = peerConnection
```

Comment:
This change seems not necessary

Reply:
just a temporary, we will remove when packages repo is ready.